docs/articles/blog/2019-2-7-migration-from-testcafe-v0-x-y-to-v1-0-0.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: Migration from TestCafe v0.x.y to v1.0.0
+permalink: /blog/:title.html
+---
+# Migration from TestCafe v0.x.y to v1.0.0
+
+TestCafe v1.0.0 release introduces minor changes to the behavior and programming interface. This document lists these changes and describes how to address them.
+
+<!--more-->
+
+* [Test Syntax Validation Disabled: All Input Files Are Executed](#test-syntax-validation-disabled-all-input-files-are-executed)
+* [Custom Request Hooks: Asynchronous API](#custom-request-hooks-asynchronous-api)
+* [Custom Reporter Plugins: Asynchronous API](#custom-reporter-plugins-asynchronous-api)
+* [Programming Interface: Multiple Method Calls Prohibited](#programming-interface-multiple-method-calls-prohibited)
+
+## Test Syntax Validation Disabled: All Input Files Are Executed
+
+Previous versions performed *test syntax validation* within input script files before executing them. Only files that contained the [fixture](../documentation/test-api/test-code-structure.md#fixtures) and [test](../documentation/test-api/test-code-structure.md#tests) directives were executed.
+
+Starting with v1.0.0, input script files are **never** validated. This means that TestCafe executes all the scripts you specify as test sources. If you use Glob patterns to specify input test files, please recheck these patterns to avoid unintended file matches.
+
+The `--disable-test-syntax-validation` command line flag and the `disableTestSyntaxValidation` option for the [runner.run](../documentation/using-testcafe/programming-interface/runner.md#run) API method that disabled test syntax validation were removed in v1.0.0.
+
+### What Has Improved
+
+You can now load tests dynamically without additional customization. The following example illustrates how tests can be imported from an external library.
+
+**external-lib.js**
+
+```js
```

Comment:
We can improve this example. If we said about dynamic test creation the following example is more accurate. We should add some dynamic data.
**external-lib.js**
```js
export default function runFixture(name, url) {
    fixture(name)
        .page(url);

    test(`${url} test`, async t => {
        // ...
    });
}
```
**test.js**
```js
import runFixture from './external-lib';

const fixtureName = 'My fixture';
const url = 'https://testPage';

runFixture(fixtureName, url);
```